test/browser/run-test.sh

```diff
@@ -73,6 +73,8 @@ EXCLUDES="$EXCLUDES
           TestMachinesNICs.NICAddDialog
 
           TestMachinesSettings.testMultipleSettings
+
+          TestMultiMachineVNC.testBasic
```

Comment:
This isn't necessary. The test isn't nondestructive, so it won't even be considered.